packages/api-gateway/src/drivers/http/routes/admin-panel/schema.js

```diff
@@ -0,0 +1,37 @@
+// TODO_JAIRO: schema doesn't work propertly
+const changeUSerStatusSchema = {
+  description: 'change user status: Given a status and a reason, change the status of a user and send a notification to the user',
+  tags: ['Admininistration panel'],
+  body: {
+    type: 'object',
+    properties: {
+      status: { type: 'string' },
+      reason: { type: 'string' },
+    },
+    required: ['status', 'reason'],
+  },
+  params: {
+    type: 'object',
+    properties: {
+      user_id: { type: 'number' },
+    },
+  },
+  response: {
+    200: {
+      type: 'string',
```

Comment:
Based on this: https://github.com/Platzi-Master-C9/booking-services/pull/160/files#r846826926
```suggestion
      type: 'object',
      properties: {
        result: { type: 'string' },
      },
```